WordPress/src/main/java/org/wordpress/android/ui/media/MediaSettingsActivity.java

```diff
@@ -745,36 +744,37 @@ private void loadImage() {
             return;
         }
 
-        if (mediaUri.startsWith("http")) {
-            showProgress(true);
-            String imageUrl = mediaUri;
-            if (SiteUtils.isPhotonCapable(mSite)) {
-                imageUrl = PhotonUtils.getPhotonImageUrl(mediaUri, size, 0);
-            }
-            mImageLoader.get(imageUrl, new ImageLoader.ImageListener() {
-                @Override
-                public void onResponse(ImageLoader.ImageContainer response, boolean isImmediate) {
-                    if (!isFinishing() && response.getBitmap() != null) {
-                        showProgress(false);
-                        mImageView.setImageBitmap(response.getBitmap());
-                        if (isMediaFromEditor()) {
-                            showImageDimensions(response.getBitmap().getWidth(), response.getBitmap().getHeight());
+        showProgress(true);
+        String imageUrl = mediaUri;
+        if (SiteUtils.isPhotonCapable(mSite)) {
+            imageUrl = PhotonUtils.getPhotonImageUrl(mediaUri, size, 0);
+        }
+        showProgress(true);
+        mImageManager.load(mImageView, ImageType.FULLSCREEN_PHOTO, imageUrl, ScaleType.CENTER,
+                new RequestListener() {
+                    @Override
+                    public void onResourceReady(@org.jetbrains.annotations.Nullable Drawable resource) {
+                        if (!isFinishing()) {
+                            if (resource != null) {
+                                showProgress(false);
+                                if (isMediaFromEditor()) {
+                                    showImageDimensions(resource.getIntrinsicWidth(), resource.getIntrinsicHeight());
+                                }
+                            } else {
```

Comment:
I think my comment in `MediaPreviewFragment` applies here as well, although it'd be slightly different.